management/src/test/java/org/ehcache/management/providers/statistics/StatsUtil.java

```diff
@@ -120,7 +120,7 @@ public static double getExpectedValueFromRatioHistory(String statName, Context c
         int mostRecentIndex = ratioHistory.getValue().length - 1;
         value = ratioHistory.getValue()[mostRecentIndex].getValue();
       }
-    } while (!Thread.currentThread().isInterrupted() && value != expectedResult);
+    } while (!Thread.currentThread().isInterrupted() && !value.equals(expectedResult));
```

Comment:
@mathieucarbou let me try your suggestion.  It would be nice to eliminate the sleep as this is causing the test to run for a really long time, and sometimes timeout.